config/enterprise_versions.yml

```diff
@@ -113,6 +113,9 @@ components:
   # The components below are third-party images that have been retagged under
   # quay.io/tigera so all enterprise images come from the same repository and org.
   elasticsearch-operator:
+    image: tigera/eck-operator
+    version: master
+  eck-elasticsearch-operator:
     image: tigera/eck-operator
```

Comment:
I think in other ones above like `eck-kibana` we don't add the image. There is a default that is set elsewhere. So maybe we could do just that.